qlik-core/engine-deployment.yaml

```diff
@@ -33,7 +33,9 @@ spec:
         args: [
           "-S", "AcceptEULA=no", # change to 'yes' if you accept the Qlik Core EULA
           "-S", "LicenseServiceUrl=http://license-service:9200",
-          "-S", "DocumentDirectory=/doc"
+          "-S", "DocumentDirectory=/doc",
+          "-S", "EnableABAC=1",
+          "-S", "SystemAllowRulePath=/rules/read_engine_rules.txt"
```

Comment:
I can't find this file in the repo. Has it been checked in?

Reply:
That file is created by the configmap in the PR ( https://github.com/qlik-oss/core-scaling/blob/14b37fc47f71b29940d29b505c70574bcf367a68/qlik-core/engine-access-control-cfg.yaml )